docs/Parameters.rst

```diff
@@ -17,18 +17,63 @@ This page contains descriptions of all parameters in LightGBM.
 Parameters Format
 -----------------
 
+Parameters are merged together in the following order (later items overwrite earlier ones):
+
+1. LightGBM's default values
+2. (CLI only) configuration in a file passed like ``config=train.conf``
+3. special files for ``weight``, ``init_score``, ``query``, and ``positions`` (see `Others <#others>`__)
```

Comment:
I believe 2 and 3 should be switched in the order.

For example, according to https://github.com/microsoft/LightGBM/blob/master/docs/Parameters.rst#weight-data
> LightGBM will load the weight file automatically if it exists.
> Also, you can include weight column in your data file.

So, `weight_column` will overwrite `train.txt.weight` file.

Reply:
ohhhhh `weight_column`, right!

I agree they should be switched in order. Did that in https://github.com/microsoft/LightGBM/pull/6579/commits/e5de60ab0674eeb1ac0e4dc03f2b201d624d2d79.